spring-integration-amqp/src/main/java/org/springframework/integration/amqp/outbound/AsyncAmqpOutboundGateway.java

```diff
@@ -41,6 +44,8 @@
 
 	private final MessageConverter messageConverter;
 
+	private volatile boolean requiresReply; // overrides super
```

Comment:
I did that at first - the problem is it causes the original async `send` (which returns null) to fail __immediately__ with a `ReplyRequiredException` :smile: